guides/common/modules/proc_upgrading-a-package.adoc

```diff
@@ -6,7 +6,8 @@ Use the following procedure to view the installed packages of a host.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the name of the host that contains the package you want to upgrade.
 . On the *Content* tab, select *Packages*.
```

Comment:
I may be blind, but I can't see a Content tab.

Reply:
Retested on Foreman 3.8:
![image](https://github.com/theforeman/foreman-documentation/assets/12595287/ce5dccaa-0220-4486-94bf-977eeb446087)